src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetail from "../../components/UserDetail";
+import UserEmail from "../../components/UserEmail";
+import UserPassword from "../../components/UserPassword";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import "./index.scss";
+
+function getArrowClass(selectedElm, currentElm) {
+  return selectedElm === currentElm ? "arrow down" : "arrow right";
+}
+
+function checkCollapseClass(selectedElm, currentElm) {
+  if (selectedElm === currentElm) return "";
+
+  return `collapsed`;
+}
+
+export default function ProfilePage({ submit }) {
+  const [selectedElement, setSelectedElement] = React.useState("userDetail");
```

Comment:
needs a comment, at this point of reading i have no idea why theres a selected element on a profile page

Reply:
I need to track the active and inactive sections to show/hide them, just show the active one and hide the others